pages/settings/profile.tsx

```diff
@@ -257,22 +302,20 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
     return { redirect: { permanent: false, destination: "/auth/login" } };
   }
 
-  const user = await prisma.user.findFirst({
-    where: {
-      email: session.user.email,
-    },
-    select: {
-      id: true,
-      username: true,
-      name: true,
-      email: true,
-      bio: true,
-      avatar: true,
-      timeZone: true,
-      weekStart: true,
-      hideBranding: true,
-    },
-  });
+  const user = await whereAndSelect(prisma.user.findFirst, {
+      id: session.user.id,
+    }, [
+      "id",
+    "username",
+    "name",
+    "email",
+    "bio",
+    "avatar",
+    "timeZone",
+    "weekStart",
+    "hideBranding",
+    "theme"
+  ]);
```

Comment:
Codacy has a fix for the issue: [Delete `]`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
  );
```